daemon/algod/api/algod.oas2.json

```diff
@@ -1035,7 +1035,7 @@
         }
       }
     },
-    "/v2/shutdown": {
+    "/v2/node/shutdown": {
```

Comment:
thank you for implementing. Technically this is API change. I guess the best way of action is:
1. to mark it deprecated (deprecated: true)
2. continue calling the shutdown method under it is old/current path along with the new one.